hw/ip/sram_ctrl/dv/cov/sram_ctrl_cov_excl.el

```diff
@@ -0,0 +1,20 @@
+// Copyright lowRISC contributors.
+// Licensed underr the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: udij
+// Format Version: 2
+// Date: Thu Jun  3 13:38:02 2021
+// ExclMode: default
+//==================================================
+CHECKSUM: "1798538324 668628424"
+INSTANCE: tb.dut.u_ram1p_sram.u_prim_ram_1p_adv.u_mem.gen_generic.u_impl_generic
+ANNOTATION: "[UNSUPPORTED]"
+Block 1 "0" "assign unused_cfg = (^cfg_i);"
+CHECKSUM: "3754679630 2080018642"
+INSTANCE: tb.dut.u_ram1p_sram
+ANNOTATION: "[UNR] init_req_q will always be high when (addr_cnt_q == Depth - 1)"
+Condition 2 "3065646248" "(init_req_q && (addr_cnt_q == (Depth - 1))) 1 -1" (1 "01")
+ANNOTATION: "[UNR] addr_cnt_q will always be less than max value when init_req_q is high"
+Condition 1 "1347804513" "(init_req_q && (addr_cnt_q < (Depth - 1))) 1 -1" (2 "10")
```

Comment:
@msfschaffner for some reason these conditions have not been caught by the UNR flow, even though they appear to be unreachable due to how the SRAM initialization counters work.
Can you provide any thoughts on this?

Reply:
As we've discussed offline, these are likely not reached due to insufficient backpressure on the TL-UL data response channel.
Once that is introduced, I would expect these to be covered.

CC @weicaiyang

Reply:
@weicaiyang what do you think RE large cycle counts in UNR?

Reply:
sorry, I'm not quite sure about this. Is this truly unreachable? or is it due to the behavior we define for `init_req_q` and the counter?

Reply:
yup - Michael and I have discussed these two conditions and we believe these are actually unreachable - Weicai has reached out to synopsys AE regarding this issue.